superdesk.config.js

```diff
@@ -30,5 +30,50 @@ module.exports = function(grunt) {
             DEFAULT_SCHEDULE: {minutes: 5, seconds: 0},
             DEFAULT_IDLE_TIME: {hours: 0, minutes: 0},
         },
+        list: {
+            priority: [
+                'priority',
+                'urgency',
+            ],
+            firstLine: [
+                'wordcount',
+                'slugline',
+                'highlights',
+                'markedDesks',
+                'associations',
+                'publish_queue_errors',
+                'headline',
+                'versioncreated',
+            ],
+            secondLine: [
+                'profile',
+                'state',
+                'scheduledDateTime',
+                'embargo',
+                'update',
+                'takekey',
+                'signal',
+                'broadcast',
+                'flags',
+                'updated',
+                'category',
+                'provider',
+                'expiry',
+                'desk',
+                'fetchedDesk',
+                'nestedlink',
+                'associatedItems',
+            ],
+            relatedItems: {
```

Comment:
I assume `relatedItems` section doesn't impact the general list. Can you move it up in the config hierarchy so it's at the same level as `list`?

Reply:
This PR is only for QA purposes, I'll take that in mind if I need to make a PR to fix something in the task